docs/setup/independent/install-kubeadm.md

```diff
@@ -51,6 +51,24 @@ The pod network plugin you use (see below) may also require certain ports to be
 open. Since this differs with each pod network plugin, please see the
 documentation for the plugins about what port(s) those need.
 
+## Installing ebtables ethtool
+
+If you see warning while running ```kubeadm init```
+
+```
+[preflight] WARNING: ebtables not found in system path                          
+[preflight] WARNING: ethtool not found in system path                           
+
+```
+You may be missing these tools on your linux, please try to install them. 
```

Comment:
suggested edit: "Then you may be missing *ebtables* and *ethtool* on your Linux machine. You can install them with the following commands:"